stable/goldilocks/templates/controller-deployment.yaml

```
@@ -26,6 +29,9 @@ spec:
         app.kubernetes.io/name: {{ include "goldilocks.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
         app.kubernetes.io/component: controller
+        {- if .Values.controller.deployment.additionalLabels }}
```

```suggestion
        {{- if .Values.controller.deployment.additionalLabels }}
```
Missing a bracket here, which may be the issue with tests.

---

I totally missed that one :) Thanks.